packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -787,6 +787,21 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
   Element get renderViewElement => _renderViewElement;
   Element _renderViewElement;
 
+  /// Schedules a [Timer] for attaching the root widget.
+  ///
+  /// This is called by [runApp] to configure the widget tree. Consider using
+  /// [attachRootWidget] if you want to build the widget tree synchronously.
+  ///
+  /// See also:
```

Comment:
nit: I would probably remove the see also here since the sentence above just said the same thing.